ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc

```diff
@@ -370,6 +370,12 @@ snpe_subplugin::configure_instance (const GstTensorFilterProperties *prop)
     cleanup ();
   }
 
+  if (!g_file_test (prop->model_files[0], G_FILE_TEST_IS_REGULAR)) {
+    std::cerr << "The file of model_path is not valid." << std::endl;
```

Comment:
please print out the given model_files[] as well.

Reply:
In addition to MJ's comment, you can use a const variable for ```"The file of model_path is not valid."```.

Reply:
I updated the code. PTAL